napari/components/_tests/test_viewer_mouse_bindings.py

```diff
@@ -58,28 +34,48 @@ def test_paint(mouse_event, modifiers, native, expected_dim):
 
     # Simulate tiny scroll
     event = ReadOnlyWrapper(
-        mouse_event(delta=[0, 0.6], modifiers=modifiers, native=native)
+        CursorEvent(
+            delta=[0, 0.6],
```

Comment:
```suggestion
            delta=(0, 0.6_),
```

Or you might get interior mutability. I'm astonished this is not caught by pydantic... or does it do that conversion itself ?

Reply:
It does the conversion itself :-)